phpDatatype/hangman.php

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Hangman</title>
+</head>
+<body>
+    <h1>Simple 'Hangman' Game</h1>
+    <h2>Guess The Band</h2>
+    <form name="hangmanForm" method="GET" action="hangman.php">
+        
+        <?php
+        global $word;
+        $word=getWord();
+        echo "<input id='word' name='word' type='hidden' value='".$word."' />";
+        totalPlaces($word);
+        ?>
+        <?php echo "<br/>"; ?>
```

Comment:
Don't open and close the PHP tags again and again. Open it once, write the PHP code and then close it. Follow this practice everywhere.